javascript/node/index.js

```diff
@@ -3,27 +3,29 @@ import {
     streamBlocks,
     createAuthInterceptor,
     createRegistry,
+    applyParams,
     fetchSubstream
 } from '@substreams/core';
 import { createConnectTransport } from "@connectrpc/connect-node";
 import { getCursor } from "./cursor.js";
 import { isErrorRetryable } from "./error.js";
 import { handleResponseMessage, handleProgressMessage } from "./handlers.js"
 
-const TOKEN = process.env.SUBSTREAMS_API_TOKEN
-const ENDPOINT = "https://mainnet.eth.streamingfast.io"
-const SPKG = "https://spkg.io/streamingfast/ethereum-explorer-v0.1.2.spkg"
-const MODULE = "map_block_meta"
-const START_BLOCK = '100000'
-const STOP_BLOCK = '+10000'
+const TOKEN = "eyJhbGciOiJLTVNFUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3NzUyMTg3MDAsImp0aSI6IjNkZDY3ODJhLWZhMTEtNGZkOS1iZGM3LTJhZDFjZTMyZGFiZCIsImlhdCI6MTczOTIxODcwMCwiaXNzIjoiZGZ1c2UuaW8iLCJzdWIiOiIwaGF2bzFiYTZkMDM0YWMxYmU5NmQiLCJ2IjoxLCJha2kiOiI0NjRlYWYxNjU0MTU4M2ZhNTNlZDg5NjAyNjdlYTc2NDQ1ZjBkYTUzNGE1ZWFiODMxMzkwYjYxMjJmNDY2MjRiIiwidWlkIjoiMGhhdm8xYmE2ZDAzNGFjMWJlOTZkIn0.aX3zE0D-YxAQTqT7Vo7vBKATOV7SB-gf0Fy5WQ80efw_o4RFLJIneL0glTSB-eM32NvlXrsraorUABB01K7Veg"
+const ENDPOINT = "https://mainnet.sol.streamingfast.io:443"
+const SPKG = "https://spkg.io/streamingfast/solana_common-v0.3.3.spkg"
```

Comment:
I'm for showcasing a parameters example, unsure if Solana should be used out of the box. Not against, but let's discuss this tomorrow.